app/workers/subscription_content_worker.rb

```diff
@@ -0,0 +1,80 @@
+class SubscriptionContentWorker
+  include Sidekiq::Worker
+
+  def perform(content_change_id:, priority:)
+    @priority = priority
+    content_change = ContentChange.find(content_change_id)
+    queue_delivery_to_subscribers(content_change)
+    queue_delivery_to_courtesy_subscribers(content_change)
+  end
+
+private
+
+  attr_reader :priority
+
+  def create_email(content_change, subscriber)
+    Email.create_from_params!(
+      email_params(content_change, subscriber)
+    )
+  end
+
+  def queue_delivery_to_subscribers(content_change)
+    subscribers_for(content_change: content_change).find_each do |subscriber|
+      begin
+        email = create_email(content_change, subscriber)
+        DeliverEmailWorker.perform_async_with_priority(
+          email.id, priority: priority
+        )
+      rescue StandardError => ex
+        Raven.capture_exception(ex, tags: { version: 2 })
+      end
+    end
+  end
+
+  def queue_delivery_to_courtesy_subscribers(content_change)
+    addresses = [
+      "govuk-email-courtesy-copies@digital.cabinet-office.gov.uk",
+    ]
+
+    Subscriber.where(address: addresses).find_each do |subscriber|
+      begin
+        email = create_email(content_change, subscriber)
+        DeliverEmailWorker.perform_async_with_priority(
+          email.id, priority: priority
+        )
+      rescue StandardError => ex
+        Raven.capture_exception(ex, tags: { version: 2 })
+      end
+    end
+  end
+
+  def subscribers_for(content_change:)
```

Comment:
I think we've lost of the use of `SubscriptionMatcher` for the various methods here.

Reply:
Yep....reinstated